### ISSUE TITLE
Is this still needed?

### ISSUE THREAD

**Issue by jbruechert** [mickybart/libhybris_ext#1]
As I understood from the repository description, this fork mainly exists for the reason you need the compat part which was removed from libhybris for some time, however it has returned back now. There are also all ubuntu changes merged with upstream libhybris now, so is there anything needed for gnulinux_support not included in upstream yet?

**Comment by mickybart** [comment]
Yes it is true but I need to rework at least 2 patches on top of upstream code and to create a pull request:

ad3b1161d1990dff51c53709632c442e7124573f
4cd30c81b303a590b22e1a85fa94ac790e58cf0a
+ maybe sf_set_power_mode in addition to sf_blank and sf_unblank

Those 2 patches will not break the compatibility and add support for 2 functions (sf_get_display_info, sf_client_get_egl_context) that are used into gnulinux_support to support SurfaceFlinger backend for kwin and QT (really specific needs mainly during the debugging time of hwcomposer backend)

**Comment by mickybart** [comment]
I update the code and created a pull request :
https://github.com/libhybris/libhybris/pull/347

If this is accepted I will update every codes and remove this repository as it will be obsolete.

EDIT: I will not reintroduce sf_set_power_mode used in this version but I will switch to sf_bank/sf_unblank. This is not important for now as Android is still using 2 states (Android 7 include)

**Comment by mickybart** [comment]
Change merged upstream so I will be able to rework the code and drop this repository.
